docker-compose.yml

```
@@ -44,7 +44,11 @@ services:
       - backend
 
   auditforge-cwe-api:
-    build: ./cwe_api
+    build:
+      context: ./cwe_api
+      dockerfile: Dockerfile
+      args:
+        FILE_ID: ${FILE_ID}     
```

yamllint

[error] trailing spaces

(trailing-spaces)